test/index.ts

```diff
@@ -174,6 +153,53 @@ pub mod a;`,
       input.dispose();
     }
   });
+
+  QUnit.test("wasm is gc'd", async (assert) => {
+    const input = await createTempDir();
+    input.write({
+      "Cargo.toml": `
+[package]
+name = "hello_lib"
+version = "0.1.0"
+authors = ["Kris Selden <kris.selden@gmail.com>"]
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.dev]
+opt-level = 1
```

Comment:
what about `z`? or is that a bad idea

Reply:
Oh I think these are just tests but in general yeah that should definitely be fine to throw in a local `Cargo.toml`!

Reply:
ah, given that they're tests, seems fine